src/arkode/arkode_mristep.c

```diff
@@ -2840,6 +2888,17 @@ int mriStepInnerStepper_FullRhs(MRIStepInnerStepper stepper, sunrealtype t,
   return stepper->last_flag;
 }
 
+int mriStepInnerStepper_FullRhsSUNStepper(MRIStepInnerStepper stepper,
+                                          sunrealtype t, N_Vector y, N_Vector f,
+                                          SUNDIALS_MAYBE_UNUSED int mode)
+{
+  SUNStepper sunstepper = (SUNStepper)stepper->content;
+  SUNErrCode err        = sunstepper->ops->fullrhs(sunstepper, t, y, f);
```

Comment:
Should the SUNStepper FullRhs also have a mode input in case it can leverage that information to avoid a function evaluation?